src/api/object.js

```diff
@@ -5,6 +5,7 @@ const DAGLink = require('ipfs-merkle-dag').DAGLink
 const promisify = require('promisify-es6')
 const bs58 = require('bs58')
 const bl = require('bl')
+const cleanMultihash = require('../clean-multihash')
 
 module.exports = (send) => {
   const api = {
```

Comment:
I can't add a comment on the old lines, but this if for the `options` part below. I learned a neat trick sometime ago for the null check: 
```javascript
const opts = Object.assign({}, options)
// use opts after this instead of options
```

and boom you don't need a if!